src/button-group/__integ__/button-group.test.ts

```diff
@@ -119,7 +120,7 @@ test(
     await page.keys(['Tab']);
     await expect(page.isFocused(likeButton.toSelector())).resolves.toBe(true);
 
-    await page.keys(['ArrowRight', 'ArrowRight']);
+    await page.keys(['ArrowRight', 'ArrowRight', 'ArrowRight', 'ArrowRight']);
```

Comment:
np: since your already used `range()` in line `55` maybe we can repeat it here too?

Reply:
I think here it is alright this way because 4 actions is easy to count anyways. I'd use range from 5-6 repeated actions personally.